### ISSUE TITLE
No limbo servers are registered, couldn't send KoxSosen to a limbo server.

### ISSUE THREAD

**Issue by KoxSosen** [McMackety/velocity-queue#7]
I can't configure the plugin correctly. When I join it says no limbo servers where registered

Velocity-Queue config:
![image](https://user-images.githubusercontent.com/67807644/103875238-77e85080-50d2-11eb-8280-9e6a875fd707.png)
Velocity Config:
![image](https://user-images.githubusercontent.com/67807644/103875290-86366c80-50d2-11eb-9d22-d162d739c6dd.png)

Issue:
![image](https://user-images.githubusercontent.com/67807644/103875317-92222e80-50d2-11eb-9af0-85d3473c104a.png)

**Comment by KoxSosen** [comment]
The issue seems to be gone after restarting the backend servers.

Is there a way I can make, so all players get sent to the limbo, than they get queued into the survival server?

**Comment by KoxSosen** [comment]
Also, if I stop the backend server, they don't get moved into the limbo server, they just get kicked from the whole proxy.

**Comment by KoxSosen** [comment]
Update: I changed my velocity config to the following:

![image](https://user-images.githubusercontent.com/67807644/103877368-3f964180-50d5-11eb-8f6d-b64d6b908c6e.png)

And now if the main server goes down, I get sent back to the limbo. But after the main server comes back I do not get sent back. Also in the limbo, I don't recive any messages regarding my position or anything about the queue, but if I join while the main server is up, i get the messages in chat.

**Comment by KoxSosen** [comment]
Is there a way that I can do the following?

Normal senario:
Player -> Limbo -> Main server
When the main server gets down:
Player -> Limbo -> Main -> Limbo
After it gets up:
Player -> Limbo -> Main -> Limbo -> Main

I don't know why I don't get any messages in the limbo, maybe because of it has many packets removed. I'm using the following limbo: https://www.spigotmc.org/resources/nanolimbo.86198/

**Comment by McMackety** [comment]
I'll add a scheduled function to automatically send players back to the server they were at on failover. Also, the first error is likely due to a network delay in pinging the servers, so let me figure out a way to fix that. I'll work on it tonight.

**Comment by KoxSosen** [comment]
Okay, thank you! Do you have any ideas why I don't get messages from the queue in the limbo, but I do in my survival? (if i set it as limbo)

**Comment by McMackety** [comment]
That's because it fails to enter you into the queue because you are being sent to the same server.

**Comment by McMackety** [comment]
If you try to switch to the same server, Velocity won't even call the ServerPreConnectEvent.

**Comment by KoxSosen** [comment]
Oh okay!

**Comment by KoxSosen** [comment]
I will wait for the next relase, and check if it's working.

**Comment by McMackety** [comment]
Sorry, I have been running low on time recently. I will try to make the change tonight to get it out of the way.

**Comment by KoxSosen** [comment]
Oh, okay.

**Comment by McMackety** [comment]
Alright, I finally got to look into it and it seems like the behaviour occurring is a simple case of undefined behaviour. You set your limbo server and initial `try = ` server to be the same, this results in Velocity throwing an error and the plugin not entering you into the queue. I started the change earlier and it should fix the issues you saw. Then on the previous error about not having any limbo server, every 5 seconds all registered servers are pinged to ensure their uptime, so wait another 5 seconds. If it still doesn't work, that means the server you registered is unreachable within your network and I cannot really help you with that, that would be an issue with your host, ip address, firewall configs, or etc. I'm going to add a message sent to the player whenever they try to connect to the server they are already on. Hopefully, this will clear up any confusion on what occurred.

Also sorry for the terrible time to test it and actually give a full response, have had a ton of large labs due in the past week.
